apps/block_scout_web/lib/block_scout_web/controllers/address_celo_controller.ex

```diff
@@ -25,7 +25,7 @@ defmodule BlockScoutWeb.AddressCeloController do
         address: address,
         current_path: current_path(conn),
         coin_balance_status: CoinBalanceOnDemand.trigger_fetch(address),
-        exchange_rate: Market.get_exchange_rate("cGLD") || Token.null(),
+        exchange_rate: Market.get_exchange_rate("CELO") || Token.null(),
```

Comment:
By the way, what was the reason behind this change https://github.com/celo-org/blockscout/commit/ed58cbf08ae816b5dc34669f0fa4945f2f752582#diff-4f6e751de6693da1e81d06fddff291aeL24? Wouldn't it be easier to have this value defined in one place?

Reply:
Hm, you know, I agree with you.  I'm not sure what the reason was as that's not my commit.

Reply:
Here's the relevant PR. https://github.com/celo-org/blockscout/pull/128  It had already been approved and had changes we needed, so I just merged once conflicts were resolved. I can ask Sami about it in the morning.

Reply:
He said it was kind of a temporary change until the token symbol has changed on-chain, but should be easy to change it to a variable